src/main/steps/applicant2/app-represented/content.ts

```diff
@@ -0,0 +1,137 @@
+  confirmationEmail: `Mae cadarnhad${
+    userCase.applicant1HelpWithFeesRefNo ? '' : ' a derbynneb am y taliad'
+  } wedi’i anfon atoch chi${isJointApplication ? ' a’ch ' + partner : ''} drwy e-bost.`,
+  subHeading6: 'Os oes arnoch angen cymorth',
+  line16: {
+    part1: 'Gall staff y llys eich helpu gyda’ch cais. Ni allant roi cyngor cyfreithiol i chi. Dylech siarad â ',
+    linkText: 'chyfreithiwr neu gynghorydd cyfreithiol',
+    link: config.get('govukUrls.legalAdvisor'),
+  },
+  webChat,
+  sendUsAMessage: 'Anfonwch neges atom ',
+  telephone: 'Ffoniwch',
+  telephoneNumber: `Rhif ffôn: ${telephoneNumber}`,
+  openingTimes,
+  telephoneCharges: {
+    part1: 'Gwybodaeth am brisiau galwadau',
+    link: config.get('govukUrls.callCharges'),
+  },
+  domesticAbuse: {
+    part1: 'Os ydych yn profi camdriniaeth ddomestig neu yn teimlo’n anniogel, yna ',
+    part2: 'mae cefnogaeth ar gael',
+    link: config.get('govukUrls.domesticAbuse'),
+  },
+  feedback: "Helpwch ni i wella'r gwasanaeth hwn",
+  feedbackLine1: 'Complete this short, 5-minute survey to help improve our services for you and others.',
+  feedbackLine2: {
+    part1: 'Mae hwn yn wasanaeth newydd.',
+    link: feedbackLink,
+  },
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const generateContent: TranslationFn = content => {
+  const { userCase, language, isJointApplication, isDivorce, isApplicant2 } = content;
+  const displayState = currentStateFn(userCase.state).at(
+    (userCase.state === State.OfflineDocumentReceived ? userCase.previousState : userCase.state) as State
+  );
+  const referenceNumber = formattedCaseId(userCase.id);
+  const isRespondentRepresented = userCase.applicant1IsApplicant2Represented === Applicant2Represented.YES;
+  const hasASolicitorContactForPartner =
+    userCase.applicant2SolicitorEmail || userCase.applicant2SolicitorAddressPostcode;
+  const isRespondentOverseas = !isCountryUk(userCase.applicant2AddressCountry);
+  const applicationServedAnotherWay =
+    !isJointApplication &&
+    userCase.applicant2Email &&
+    !isRespondentOverseas &&
+    !userCase.iWantToHavePapersServedAnotherWay &&
+    !hasASolicitorContactForPartner;
+  const cannotUploadDocuments = new Set([
+    ...(userCase.applicant1CannotUploadDocuments || []),
+    ...(userCase.applicant2CannotUploadDocuments || []),
+  ]);
+  const progressBarContent = getProgressBarContent(isDivorce, displayState, language === SupportedLanguages.En);
+  const feedbackLink = `${config.get('govukUrls.feedbackExitSurvey')}/?service=${
+    isDivorce ? 'Divorce' : 'Civil'
+  }&party=${isJointApplication ? (isApplicant2 ? 'jointapp2' : 'jointapp1') : 'app'}`;
+  return {
+    ...languages[language]({ ...content, referenceNumber }, feedbackLink),
+    displayState,
+    isRespondentRepresented,
+    hasASolicitorContactForPartner,
+    isRespondentOverseas,
+    applicationServedAnotherWay,
+    referenceNumber,
+    cannotUploadDocuments,
+    ...progressBarContent,
+  };
```

Comment:
We might be able to reuse the content/template for app 1 in content.ts and template.njk. The hub page for app 2 is a good example:
https://github.com/hmcts/nfdiv-frontend/blob/1a2f77026f34db78eddf8f5611696e76113bb767/src/main/steps/applicant2/hub-page/content.ts#L5

Reply:
I don't like that we do this tho - we should have common code in a common not under 1. But I couldn't face refactoring, so maybe I will re-use